velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -922,6 +922,19 @@ struct DateTruncFunction : public TimestampWithTimezoneSupport<T> {
     }
   }
 
+  // For fixed interval like second, minute, hour, day and week
+  // we can truncate date by a simple arithmetic expression:
+  // floor(seconds / intervalSeconds) * intervalSeconds
```

Comment:
nit: use /// for multiple-line comments of a public API, and add . at the end for comments.